src/exchanges/index.ts

```diff
@@ -2,6 +2,7 @@ export { cacheExchange } from './cache';
 export { subscriptionExchange } from './subscription';
 export { dedupeExchange } from './dedup';
 export { fetchExchange } from './fetch';
+export { composeExchanges } from './compose';
```

Comment:
This should make it easier for people to compose some existing exchanges—be them internal to `urql` or third-party—outside of urql as well, in case someone writes tests or even a library that needs some of our exchanges to work. In such a case someone can use `composeExchanges` instead of exposing an exchange array, so hopefully a nice addition